templates/gem/test/minitest_helper.rb

```diff
@@ -0,0 +1,44 @@
+=begin
+Copyright <%= copyright %>
+
+See the file LICENSE for copying permission.
+=end
+
+if
+  RUBY_VERSION == "2.0.0" # check Gemfile
+then
+  require "coveralls"
+  require "simplecov"
+
+  SimpleCov.start do
+    formatter SimpleCov::Formatter::MultiFormatter[
+      SimpleCov::Formatter::HTMLFormatter,
+      Coveralls::SimpleCov::Formatter,
+    ]
+    command_name "Spesc Tests"
+    add_filter "/test/"
+  end
+
+  Coveralls.noisy = true unless ENV['CI']
+end
+
+require 'minitest/autorun'
+require "minitest/reporters"
+require "mocha/setup"
+require 'pluginator'
+
+module PreCommit; module Helpers
+
+  def project_dir
+    File.expand_path("../files", __FILE__)
+  end
```

Comment:
I don't think this is `project_dir`.

Maybe this is `project_dir`:

```ruby
puts File.expand_path("../..", __FILE__)
```

Or we could rename this method to `test_files`, `fixtures_dir`, `fixture_files`?